model/src/main/java/com/javahelp/model/user/ProviderUserInfo.java

```diff
@@ -10,7 +10,7 @@ public class ProviderUserInfo extends UserInfo {
     private String phoneNumber;
 
     /**
-     * Constructs a ProviderInfo object.
+     * Constructs a ProviderInfo object without given gender
```

Comment:
Gender should be removed from this PR entirely, it should be entirely in its own PR.

Reply:
Sorry this is a misleading docstring